dp-terraform/helm/rhacs-terraform/values.yaml

```
@@ -91,3 +91,78 @@ logging:
   aws:
     accessKeyId: ""
     secretAccessKey: ""
+
+vector:
+  role: "Aggregator"
+  service:
+    annotations:
+      rhacs.redhat.com/cluster-name: ""
+      rhacs.redhat.com/environment: ""
+      service.beta.openshift.io/serving-cert-secret-name: rhacs-vector-tls-secret
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/name
+                operator: In
+                values:
+                  - vector
+          topologyKey: "topology.kubernetes.io/zone"
+  persistence:
+    enabled: true
+    size: 100Mi
+  replicas: 3
+  extraVolumes:
+    - name: service-tls-secret
+      projected:
+        sources:
+          - secret:
+              name: rhacs-vector-tls-secret
+  extraVolumeMounts:
+    - name: service-tls-secret
+      mountPath: /etc/vector/tls
+      readOnly: true
+  customConfig:
+    sources:
+      http_server:
+        type: "http_server"
+        address: "0.0.0.0:8888"
+        decoding:
+          codec: "json"
+        tls:
+          enabled: true
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+          crt_file: "/etc/vector/tls/tls.crt"
+          key_file: "/etc/vector/tls/tls.key"
+    sinks:
+      aws_s3:
+        type: "aws_s3"
+        region: ""
+        bucket: ""
+        key_prefix: '{{ "{{" }} .tenant_id {{ "}}" }}/%F/'
+        inputs: ["http_server"]
+        compression: none
+        filename_extension: "json"
+        healthcheck:
+          enabled: false
+        batch:
+          timeout_secs: 10
```

I have looked a bit at the configuration of helm:
https://github.com/vectordotdev/helm-charts/blob/develop/charts/vector/README.md

And S3 sink:
https://vector.dev/docs/reference/configuration/sinks/aws_s3/#buffer.max_size

From what I see, enabling `persistence` will create PV/PVC, but it will not configure the application to use it. So we have to configure Vector accordingly to use that persistence.

Also, size of the buffer (including disk) has to be at least 256MB - https://vector.dev/docs/reference/configuration/sinks/aws_s3/#buffer.max_size

My proposal would be:
- 270 MB buffer - 300 MB disk
- sync every 1 min
- max batch size 2.5MB
- if buffer is full, block requests -> this should pop-up errors in central

```suggestion
        batch:
          timeout_secs: 60
          max_bytes: 2621440
        buffer:
          type: "disk"
          max_size: 283115520
          when_full: "block"
```